programming/arduino/index.md

```diff
@@ -3,134 +3,21 @@ layout: page
 title: Arduino API
 ---
 
-Arduino API
-===========
+# Arduino API
 
-The [Ruggeduino](https://web.archive.org/web/20170317171649/https://www.rugged-circuits.com/ruggeduino)
-provides a total of 18 pins for either digital input or output (labelled 2 to 13 and A0 to A5),
-including 6 for analogue input (labelled A0 to A5).
+The Arduino that came as part of your kit was shipped with a firmware that provides the functionality outlined in the [Arduino API](/docs/programming/arduino/sr_firmware) page.
```

Comment:
Suggestion: We should explicitly note here that using other arduinos is entirely supported (some may assume ours is _magic_)

Reply:
33a3875